sasmodels/models/lib/gauss150.c

```diff
@@ -6,9 +6,19 @@
  *  Copyright 2007 __MyCompanyName__. All rights reserved.
  *
  */
+ #ifdef GAUSS_N
+ # undef GAUSS_N
+ # undef GAUSS_Z
+ # undef GAUSS_W
+ #endif
+ #define GAUSS_N 150
+ #define GAUSS_Z Gauss150Z
+ #define GAUSS_W Gauss150Wt
+
+// Note: using array size 152 so that it is a multiple of 4
 
 // Gaussians
-constant double Gauss150Z[150]={
+constant double Gauss150Z[152]={
```

Comment:
You might want to document why the extra zeros are added so no one tries to remove them in the future.

Reply:
Done:
{{{
// Note: using array size 152 rather than 150 so that it is a multiple of 4.
// Some OpenCL devices prefer that vectors start and end on nice boundaries.
}}}